src/Microsoft.DotNet.Interactive.Tests/Notebook/JupyterNotebookDocumentFileFormatTests.cs

```diff
@@ -112,6 +112,89 @@ public void missing_metadata_defaults_to_csharp_kernel()
                 });
         }
 
+        [Fact]
+        public void cell_metadata_can_specify_language_that_overrides_notebook()
+        {
+            var jupyter = new
```

Comment:
Why are we using anonymous types for this sort of thing?

Reply:
This structure is really json, but I didn't want to deal with all of the quote escaping so I did it this way when I first added the notebook parser a few months ago.

Reply:
I discussed this with @colombod and he's in agreement that anonymous types are fine here because we really don't want to be using strong typing because that misses the point of the input to the function we're actually testing being able to have many different shapes (e.g., a cell's source can either be a single string or an array of strings, and we can't model that with strong types, but we have to be able to test both.)